web/src/condense.ts

```diff
@@ -32,15 +33,31 @@ const state: {
     has_unread_cutoff_by_message_id: new Set<number>(),
 };
 
-export function show_message_expander($row: JQuery): void {
+export function show_message_expander(
+    $row: JQuery,
+    tooltip_template_id: string | null = "message-expander-tooltip-template",
+    toggle_button_label: string = $t_html({defaultMessage: "Show more"}),
```

Comment:
You should use `$t` and `{{` and use it as text, rather than HTML, unless it actually needs HTML.

Reply:
`label_text` is the right variable name for something that's not HTML; I guess I'll fix that before merging.